sentences/cs/light_HassLightSet.yaml

```diff
@@ -11,8 +11,40 @@ intents:
           - "[jas] <area> (<zvysit>|<ztlumit>|<nastavit>|<zmenit>) [jas] <brightness>"
         slots:
           name: "all"
+
+        #Max/Min sentences
+      - sentences:
+          - "(<nastavit>|<zvysit>|<zmenit>) [jas] <area> na [co] (maximum | max | nejvíc)"
+          - "(<nastavit>|<zvysit>|<zmenit>) [na] [co] (maximální | nejvyšší | největší) [jas] <area>"
+          - "[jas] <area> (<zvysit>|<nastavit>|<zmenit>) [jas] [na] [co] [(nejvyšší|maximální|nejvíc)] [jas] [(maximum | max)]"
```

Comment:
[co] seems to be related only to nejmíň, don't you think?

Reply:
yep, good point